charts/camunda-platform/values.yaml

```diff
@@ -1754,7 +1754,26 @@ connectors:
   # Switch for inbound mode (e.g., for webhook or polling)
   inbound:
     # Acceptable values: disabled, credentials, or oauth
-    mode: disabled
+    mode: oauth
+
+    # Operate client configurations
+    operateClient:
```

Comment:
Am I correct, that you suggest to:

- Leave `inbound.mode` in the `values.yaml`
- Move the `inbound.operateClient.*` to a `kind: Secret`

?

Reply:
yes, moving `inbound.operateClient.*` in secret, and using it in the corresponding env vars.

for example, the cred vars credentials, instead doing this:
```
- name: CAMUNDA_OPERATE_CLIENT_USERNAME
  value: {{ .Values.connectors.inbound.operateClient.username }}
- name: CAMUNDA_OPERATE_CLIENT_PASSWORD
  value: {{ .Values.connectors.inbound.operateClient.password }}
```

it will be:
```
- name: CAMUNDA_OPERATE_CLIENT_USERNAME
  valueFrom:
    secretKeyRef:
      name: "{{ default "camunda-platform-connectors" .Values.connectors.clients.existingSecret }}"
      key: username
- name: CAMUNDA_OPERATE_CLIENT_PASSWORD
  valueFrom:
    secretKeyRef:
      name: "{{ default "camunda-platform-connectors" .Values.connectors.clients.existingSecret }}"
      key: password
```

in the same ,secret you can handle the `operateOAuthClientId` and `operateOAuthClientSecret` keys too.
so here, first, we don't store secrets in the values
second, the users can provide their credentials using `.connectors.clients.existingSecret`.
thierd, the same secret could be used in Identity instead of duplicating the values as in 
[charts/camunda-platform/charts/identity/templates/deployment.yaml](https://github.com/camunda/camunda-platform-helm/pull/583/files/38b4227068719f7c81973d6029dec57e9aa55b64#diff-0211d5340edc0f26bec547031979cef5350ce0d5cd86a602e4f75108de1810feR120).

if anything is not clear, let's talk on Slack.

Reply:
Done in a bit different way, please review. Also left a note in Slack.

Reply:
I mark this as done since I implemented the auth the same way as the other components.